### ISSUE TITLE
Request to Dev: Tutorials for RK3566 OrangePi 3b

### ISSUE THREAD

**Issue by defencedog** [ProKn1fe/SBC_Playground#1]
I have Opi3b v2.1 & I intend to compile `Bitfrost MaliG52` gpu userspace binary driver. 

To that extent I already have made `panfrost` capable kernel which is properly detected in `ubuntu-gnome` session under `X11` ... currently I cannot get `wayland` to work with `legacy` kernel 5.1.xx

My progress: https://github.com/defencedog/orangepi3b_v2.1/tree/main

**Comment by ProKn1fe** [comment]
Why legacy 5.10 kernel and not current 6.1?
Also better to use patched kernel instead of very old orangepi. I recommend joshua or armbian kernel tree https://github.com/Joshua-Riek/linux-rockchip (noble branch is 6.1 and jammy is 5.10).

**Comment by defencedog** [comment]
@ProKn1fe because of hardware issues that still are not sorted out in main-line Kernels. I as an engineer needs Bluetooth & 3.5mm audio jack output....

**Comment by ProKn1fe** [comment]
@defencedog those problems are in edge 6.6+ kernel, 6.1 vendor kernel is from rockchip.

**Comment by defencedog** [comment]
@ProKn1fe kindly guide please. [Armbian official page](https://www.armbian.com/orangepi3b/) for Opi3b don't provide `vendor` kernel anymore & I cannot find any archive release or even archive links? If you have do please provide

**Comment by ProKn1fe** [comment]
@defencedog build yourself. Opi3b in armbian in unsupported state.
![image](https://github.com/user-attachments/assets/2ea8a956-fd6e-48cb-aefc-c0acd6c3b3b2)
![image](https://github.com/user-attachments/assets/709c6498-70ab-4f67-ab90-e776a1dbe278)

**Comment by defencedog** [comment]
@ProKn1fe I don't have `5.1.xx` option, secondly I built no-desktop server image using `6.1` kernel but it gave me `kernel panic `  I am using `Win10 WSL2`

Using `find . | grep` commands I see there is no `orangepi3v-v2.dtb` file

![Screenshot_7](https://github.com/user-attachments/assets/4271c2e0-9aa2-4d15-8662-6397681ada9b)
[Armbian-unofficial_24.8.0-trunk_Orangepi3b_jammy_vendor_6.1.75putty.log](https://github.com/user-attachments/files/16532274/Armbian-unofficial_24.8.0-trunk_Orangepi3b_jammy_vendor_6.1.75putty.log)

I would be grateful if you ccan help directly or give me some further instructions

**Comment by defencedog** [comment]
Additional note I recently downloaded official image from Armbian website & same kernel panic; I believe they have not added sources for `Opi3b v2.1`

[Armbian_community_24.8.0-trunk.369_Orangepi3b_noble_vendor_6.1.43_gnome_desktop_putty.log](https://github.com/user-attachments/files/16532294/Armbian_community_24.8.0-trunk.369_Orangepi3b_noble_vendor_6.1.43_gnome_desktop_putty.log)

[Armbian_community_24.8.0-trunk.495_Orangepi3b_noble_vendor_6.1.75_gnome_desktop_putty.log](https://github.com/user-attachments/files/16532297/Armbian_community_24.8.0-trunk.495_Orangepi3b_noble_vendor_6.1.75_gnome_desktop_putty.log)

**Comment by ProKn1fe** [comment]
It will not boot at all if it do not have dts file. It is https://github.com/armbian/linux-rockchip/blob/rk-5.10-rkr6/arch/arm64/boot/dts/rockchip/rk3566-orangepi-3b.dts

Do this images boots at all? https://github.com/Joshua-Riek/ubuntu-rockchip

**Comment by defencedog** [comment]
@ProKn1fe Yes it does, however Bluetooth doesn't work. I am a process engineer this is main requirement because I have to communicate with field devices.

Recently I tried Armbian image which had Panfrost by default, it too didn't have BT; WIFI however works. Its weird because WIFI / BT are supplied by same chip

See the last logs when I triend enabling BT via gnome radio button

[Armbian_community_24.8.0-trunk.495_Orangepi3b_noble_edge_6.10.2_gnome_putty.log](https://github.com/user-attachments/files/16534115/Armbian_community_24.8.0-trunk.495_Orangepi3b_noble_edge_6.10.2_gnome_putty.log)
